packages/neuron-wallet/src/services/wallets.ts

```diff
@@ -371,18 +371,55 @@ export default class WalletService {
     const paths = addressInfos.map(info => info.path)
     const pathAndPrivateKeys = this.getPrivateKeys(wallet, paths, password)
 
-    const witnesses: string[] = inputs!.map((input: Input) => {
+    const witnessesWithLockHashes = inputs!.map((input: Input) => {
```

Comment:
`witnessesWithLockHashes` is really a bad name. If I understand how it was named previously, it should be called `witnessesWithLockHashesAndWitnessArgsAndBlake160` now?